docusaurus.config.js

```diff
@@ -233,7 +233,7 @@ module.exports = {
           items: [
             {
               type: "doc",
-              docId: "tools/tools-1-0-0",
```

Comment:
Should this have `pre` as well?

Reply:
No, this is going by docId vs name/label which doesn't have `pre` :)

Reply:
Good to know, thanks!